src/main/java/com/inventorysystem/inventory/dao/RetrofitClientInstance.java

```diff
@@ -0,0 +1,22 @@
+package com.inventorysystem.inventory.dao;
+
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitClientInstance {
```

Comment:
did a base setup for retrofit since you need to get the data from the endpoint eventually